unittests/test_environments.py

```diff
@@ -248,6 +259,33 @@ def test_emit_loadenv_commands(base_environ, user_runtime,
                                modules_system, env0):
     ms = rt.runtime().modules_system
     expected_commands = [
+        *env0.prepare_cmds,
+        ms.emit_load_commands('testmod_foo')[0],
+        'export _var0=val1',
+        'export _var2=$_var0',
+        'export _var3=${_var1}',
+    ]
+    assert expected_commands == rt.emit_loadenv_commands(env0)
+
+
+def test_emit_loadenv_nomod_commands(base_environ, user_runtime, env0):
```

Comment:
Do we need this test? I think it's the same as the `test_emit_loadenv_commands`, which btw is defined twice. Probably a copy-paste error?

Reply:
I needed it so that I test when no modules system is defined. I added it to be able to test locally.